tests/artifactorysecurity_test.go

```diff
@@ -136,6 +136,42 @@ func getTokensTest(t *testing.T) {
 	tokensToRevoke = append(tokensToRevoke, token.RefreshToken)
 }
 
+func getUserTokenstest(t *testing.T) {
```

Comment:
Let's make sure that the test revokes the tokens it generates. See the ```tokensToRevoke``` slice used in other tokens tests.

Reply:
Done.